ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py

```diff
@@ -1292,3 +1292,40 @@ def test_torch_diag(
         input=values[0],
         diagonal=diagonal,
     )
+    
+ 
+@handle_frontend_test(
+    fn_tree="torch.bucketize",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shared_dtype=True,
+        min_num_dims=1,
+        max_num_dims=1,
+    ),
+    boundaries=st.lists(st.integers(1, 10), min_size=1, max_size=10)),

```

Comment:
`boundaries` was specified by torch's documentation as:

> 1-D tensor, must contain a strictly increasing sequence, or the return value is undefined.

But `st.lists(...)` doesn't necessarily generate an increasing sequence, it is a good approach to exhaustively test your function, but the probability that the strategy does generate a valid increasing sequence is very low as it randomly samples numbers from a list of integers. So all our test cases might be falling under the _"return value is undefined"_ criterion.